plugins/modules/container_remote.py

```diff
@@ -75,11 +75,11 @@
 """
 
 RETURN = r"""
-  remotes:
+  pulp.squeezer.remotes:
```

Comment:
Return values have FQCNs now? ;)

Reply:
:facepalm:

Reply:
I would not risk a bet against it though.